internal/api/testapi/testapi.go

```diff
@@ -0,0 +1,385 @@
+package testapi
+
+import (
```

Comment:
Suggestion: Please add some tests for the high level client. The tests might even replace the scripts obsolete at some point. The benefit is just that tests keep running while the scripts at some point might unknowingly diverge from the code.